### PR TITLE
Add iteration 4 details and gif

### PR DESCRIPTION
**Pull request by jennadilullo8**
**Add Iteration 4**

- Added the details of what has been completed in Iteration 4.
- Added a gif to demonstrate what has been completed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,3 +82,12 @@ Gif of application collapsing to mobile layout with and without menu bar:
 - All of this is completed without reloading the page
 
 <img src="http://g.recordit.co/lKJIm2RusO.gif" alt="delete, star, and active star button functionality" height=auto width=75%/>
+
+### Iteration 4 - Local Storage & Filtering
```
Comment (lilianaweimer) [inline README.md:86]: Great description of the iteration!

File: README.md [hunk 2]
```diff
@@ -88,3 +82,12 @@ Gif of application collapsing to mobile layout with and without menu bar:
...
+
+- When the user refreshes the page, the idea cards will be displayed.
+- When one of those idea cards are deleted and the page is then refreshed, that idea card will no longer show.
```
Comment (asodemann18) [inline README.md:89]: should be "is deleted"

**Review by lilianaweimer — APPROVED** [review]
Looks good! :)

File: README.md [hunk 3]
```diff
@@ -88,3 +82,12 @@ Gif of application collapsing to mobile layout with and without menu bar:
...
+- When the user clicks on the "star" button on an idea card, and then refresh the page, the idea card will still have the "star" button highlighted
```
Comment (asodemann18) [inline README.md:90]: should say "and then _refreshes_ the page"

**Comment by asodemann18** [comment]
Looks good! I made some comments on the individual lines to fix some small grammar things.

**Pull request merged**